exporter/awsemfexporter/config.go

```diff
@@ -64,6 +64,11 @@ type Config struct {
 	// TODO: we can support directing output to a file (in the future) while customer specifies a file path here.
 	OutputDestination string `mapstructure:"output_destination"`
 
+	// CreateEKSFargateKubernetesObject is an option to reformat certin metric labels so that they take the form of a high level object
```

Comment:
```suggestion
	// EKSFargateContainerInsightsEnabled is an option to reformat certin metric labels so that they take the form of a high level object
```

Ensure the rest of this comment matches the current implementation.